README.md

```diff
@@ -1,24 +1,26 @@
-<p align="center"><a href="http://www.codechefvit.com" target="_blank"><img src="https://s3.amazonaws.com/codechef_shared/sites/all/themes/abessive/logo-3.png" title="CodeChef-VIT" alt="Codechef-VIT"></a>
+<p align="center"><a href="http://www.codechefvit.com" target="_blank">
+<img src="https://s3.amazonaws.com/codechef_shared/sites/all/themes/abessive/logo-3.png" title="CodeChef-VIT" alt="Codechef-VIT"></a>
 </p>
 
 # LeoClub-Website
 Official website for the Leo Club VIT.
 
 ---
 [![DOCS](https://img.shields.io/badge/Documentation-see%20docs-green?style=flat-square&logo=appveyor)](https://github.com/CodeChefVIT/LeoClub-Website) 
-[![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](INSERT_UI_LINK_HERE)
+[![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](https://github.com/CodeChefVIT/LeoClub-Website/blob/master/index.html)
 
 ## Features
-- Website is made by using HTML , CSS and Bootstrap.
+- Website is made by using HTML , CSS , Jquery and Bootstrap.
 - Contains information about LeoClub VIT , their previous events and their current board and committe members.
 
 ## Screenshots
-<img src="https://github.com/Aarushi21/LeoClub-Website/tree/master/assets/img/screenshot" alt="Project Screenshots">
+<img src="https://github.com/CodeChefVIT/LeoClub-Website/blob/master/assets/img/screenshot.png" alt="Project Screenshot">
```

Comment:
@yasharthratan Please use markdown syntax.

Reply:
Any specific reason for using `HTML` tags?
Is it the part of the template provided?

Reply:
@vinitshahdeo 

Yes we have used HTML tags in the official template for images (to assign custom dimensions) and to <p> with center attribute to center stuff.

Reply:
Cool 👍

I appreciate your work. You guys are doing great. Keep it up:)

Reply:
@vinitshahdeo thank you 😊